docs/configuration.md

```diff
@@ -76,14 +72,19 @@ $ oc create clusterrolebinding wasp --clusterrole=cluster-admin --serviceaccount
 $ oc adm policy add-scc-to-user -n wasp privileged -z wasp
 ```
 
-4. Wait for the worker nodes to sync with the new config:
+6. #### Wait for the worker nodes to sync with the new config:
 ```console
 $ oc wait mcp worker --for condition=Updated=True --timeout=-1s
 ```
 
-5. Deploy `wasp-agent`
-   Create a `DaemonSet` according to the following
-   [example](../manifests/openshift/ds.yaml).
+7. #### Deploy `wasp-agent` <br>
+ * ##### Determine wasp-agent image pull URL:
+```console
+$ OCP_VERSION=$(oc get clusterversion | awk 'NR==2' |cut -d' ' -f4 | cut -d'-' -f1)
+$ oc get csv kubevirt-hyperconverged-operator.v${OCP_VERSION} -nopenshift-cnv -ojson | jq '.spec.relatedImages[] | select(.name|test(".*wasp-agent.*")) | .image'
+"registry.redhat.io/container-native-virtualization/wasp-agent-rhel9@sha256:c1f49d460918baa3cc5340ed7ff274c0a32df161310a7f01a6704ffbe348d23b"
```

Comment:
I'm concerned that including the full image URL might confuse users. I'm not sure it's necessary.